ariadne/asgi.py

```diff
@@ -58,6 +66,8 @@ def __init__(
         self,
         schema: GraphQLSchema,
         *,
+        on_connect: Optional[OnConnect] = None,
+        on_close: Optional[OnClose] = None,
```

Comment:
```suggestion
        on_ws_connect: Optional[OnConnect] = None,
        on_ws_close: Optional[OnClose] = None,
```
The name should represent that this only belongs to the websocket connection.

Reply:
Why? Are there other connection types that GraphQL is aware of that could be confusing?